src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -595,6 +596,20 @@ def load_command_table(self, _):
         g.custom_command('enable', 'server_aad_only_enable')
         g.show_command('get', 'get')
 
+    ###############################################
+    #           sql server trust groups           #
+    ###############################################
+
+    server_trust_groups_operations = CliCommandType(
+        operations_tmpl='azure.mgmt.sql.operations#ServerTrustGroupsOperations.{}',
+        client_factory=get_sql_server_trust_groups_operations)
+
+    with self.command_group('sql stg', server_trust_groups_operations, client_factory=get_sql_server_trust_groups_operations, is_preview=True) as g:
+        g.custom_command('create', 'server_trust_group_create', supports_no_wait=True)
+        g.custom_command('delete', 'server_trust_group_delete', confirmation=True, supports_no_wait=True)
+        g.custom_show_command('show', 'server_trust_group_get')
+        g.custom_command('list', 'server_trust_group_list')
```

Comment:
Could you add wait command according to the guidance?
![image](https://user-images.githubusercontent.com/49508232/111560410-5f30a100-87cd-11eb-8a68-980785dc555d.png)